docs/user/02-logs.md

```diff
@@ -320,6 +307,22 @@ spec:
   ...
 ```
 
+### Result
+
+To activate and verify the constructed LogPipeline, follow these steps:
```

Comment:
Isn't activating the LogPipeline rather something like step 7? Why construct it if you're not going to activate it? 
In that case, I'd put steps 1+2 of Result as a last step 7 of the procedure above. And then the verification step would be part of the Result section, along the lines of: "Result: Your Logpipeline has been activated and has the status `Ready` in your cluster. Check the status with the following command:"

Reply:
done